qiskit/transpiler/preset_passmanagers/level0.py

```diff
@@ -189,13 +191,17 @@ def _choose_layout_condition(property_set):
         optimization = plugin_manager.get_passmanager_stage(
             "optimization", optimization_method, pass_manager_config, optimization_level=0
         )
+    post_translation = None
+    if pass_manager_config.post_translation_pm is not None:
+        post_translation = pass_manager_config.post_translation_pm
```

Comment:
```suggestion
    post_translation = pass_manager_config.post_translation_pm
```
The `if` seems unnecessary?  (etc in other presets)

Reply:
Yeah, that's a good point. I'll update it

Reply:
Superseded by https://github.com/Qiskit/qiskit-terra/pull/8648/commits/cf6a7ec90ab8b8b150abb92cce15a7935a0b6994